modules/migration-combining-must-gather.adoc

```diff
@@ -9,7 +9,7 @@ Currently, it is not possible to combine must-gather scripts, for example specif
 
 [source,terminal]
 ----
-$ oc adm must-gather --image=brew.registry.redhat.io/rh-osbs/oadp-oadp-mustgather-rhel8:1.1.1-8  -- skip_tls=true /usr/bin/gather_with_timeout <timeout_value_in_seconds>
+oc adm must-gather --image=registry.redhat.io/oadp/oadp-mustgather-rhel9:v1.3 -- skip_tls=true /usr/bin/gather_with_timeout <timeout_value_in_seconds>
```

Comment:
I've falling into a few traps with stating exact versions in a code snippet instead of using a user-replaced value. Would user-replaced values work better instead of stating the RHEL version? Maybe an example could be provided with a callout but places the onus on a customer to check a version of the tool is compatible with their infrastructure?

Reply:
@dfitzmau 
I think this is beyond the scope of this PR.